fe4u-boilerplate-main/src/js/Popup/popupaddTeacher.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+const Validator = require('../RawDate/userValidator');
+const {createUser} = require('../processRawDate');
+
+class PopupAddTeacher {
+     FormListener(callback) {
+        this.popupForm.onsubmit = (event) => {
+            this.teacherData = "";
```

Comment:
Not used variable. Use WebStorm (free for students), it shows such mistakes from the box.